### PR TITLE
Native YAML - files/file.py

### PR DESCRIPTION
**Pull request by Fale**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
files/file.py

##### ANSIBLE VERSION
```
2.2
```

##### SUMMARY
Make YAML code use the native style

@gundalow

### CODE REVIEW

**Review by gundalow — COMMENTED** [review]
Unsure about with_items rewrite

File: files/file.py [hunk 1]
```diff
@@ -100,12 +100,14 @@
     group: foo
     state: link
 - file:
-    src: /tmp/{{ item.src }}
-    dest: "{{ item.dest }}"
+    src: '/tmp/{{ item.src }}'
+    dest: '{{ item.dest }}'
     state: link
   with_items:
-    - { src: 'x', dest: 'y' }
```
Comment (gundalow) [inline files/file.py:107]: I'm going to say that rewriting the `write_items` section here decreases readability, can you please revert that bit.

**Review by gundalow — CHANGES_REQUESTED** [review]
needs_revision

**Comment by gregdek** [comment]
Thanks @Fale for this PR. This module is maintained by the Ansible core team, so it can take a while for patches to be reviewed. Thanks for your patience.

Core team: please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with 'needs_revision' or merge as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Review by gundalow — APPROVED** [review]

**Pull request merged**